app/controllers/notifications_controller.rb

```diff
@@ -30,4 +29,14 @@ def show
       end
     end
   end
+
+  def mark_read
```

Comment:
We should probably do this via REST instead of RPC, since that's the convention in both ED and Rails.  Ember-Data can set `seen: true` and then `.save()` the instance.  "Mark all read" could be done by issuing an update to a notification with magic id `all` — you might have to do a direct AJAX call but ED could handle the individual ones.